Serialize/Services.cs

```diff
@@ -1128,15 +1166,6 @@ public static void EncodeEventNotifyUnconfirmed(EncodeBuffer buffer, BacnetEvent
             EncodeEventNotifyData(buffer, data);
         }
 
-        public static void EncodeAlarmSummary(EncodeBuffer buffer, BacnetObjectId objectIdentifier, uint alarmState, BacnetBitString acknowledgedTransitions)
-        {
-            /* tag 0 - Object Identifier */
-            ASN1.encode_application_object_id(buffer, objectIdentifier.type, objectIdentifier.instance);
-            /* tag 1 - Alarm State */
-            ASN1.encode_application_enumerated(buffer, alarmState);
-            /* tag 2 - Acknowledged Transitions */
-            ASN1.encode_application_bitstring(buffer, acknowledgedTransitions);
-        }
```

Comment:
@LADSoft what is the reason to delete this part?

Reply:
I think I thought I had added it, and wasn't going to be able to test it.    But if it was already there I could put it back in...

Reply:
Ok, so I suggest to put it back in, thank you. I know the original codebase is a mess, but as long as we don't rewrite it, it makes it easier to "cherry pick" fixes from hte original codebase if we don't change it too much.

Reply:
I added a new commit to readd the function...